src/sagemaker/jumpstart/model.py

```diff
@@ -301,6 +302,10 @@ def __init__(
                 optionally applied to the model.
             additional_model_data_sources (Optional[Dict[str, Any]]): Additional location
                 of SageMaker model data (default: None).
+            accept_draft_model_eula (bool): For draft models that require a Model Access Config, specify True or
```

Comment:
Why do we have to introduce a new parameter here ?

The draft model source is part of `additional_model_data_sources `right ? We should be able to directly set `ModelAccessConfig `in `additional_model_data_sources  ` wherever required ?

Reply:
Have clarified the requirements with Loki, iterating on feedback